docs/primer.md

```diff
@@ -262,6 +262,18 @@ In addition an element has the optional properties `notNull`, and `default` (bot
 
 Note: As a general convention we recommend to omit optional boolean properties, if the value is `false`.
 
+### Cardinality
+
+Please note that CSN Interop does not support arrays of scalar types, only of [cds.Association](./spec-v1/csn-interop-effective#association-type) and [cds.Composition](./spec-v1/csn-interop-effective#composition-type), which can express this in their [cardinality](./spec-v1/csn-interop-effective#cardinality-object) object.
+
+The reason is that arrays of scalar types are not supported in all tech-stacks and would require a more complex mapping.
+By not supporting this in CSN Interop, the model needs to make the decision how multi-value scalar types are to be mapped.
+
+This can be done, e.g. by:
+
+- converting the scalar type to an Entity and use `cds.Association` or `cds.Composition` with the appropriate cardinality (RECOMMENDED).
+- converting the array of values into a single string, e.g. by using a delimiter or parsing it to a JSON string.
```

Comment:
If this is done, it would be consequent to also have a `@Semantics` annotation to indicate this. E.g. `@Semantics.json` or `@Semantics.commaSeparated`, `@Semantics.semicolonSeparated`